src/util/tests/getDeriveAccount.test.ts

```diff
@@ -17,6 +17,8 @@
 import getDeriveAccount from '../getDeriveAccount';
 
 describe('getDeriveAccount', () => {
+  process.env.REACT_APP_ACCOUNT_DERIVATION = 'pallet-bridge/account-derivation/account';
```

Comment:
This variable is not used any more - and that's correct, cause that's not something that can be configured. Please remove it from `.env`, `.env.example` and here.